src/main/webapp/src/app/recommendation-form/recommendation-form.component.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+
+import { RecommendationFormComponent } from './recommendation-form.component';
+
+describe('RecommendationFormComponent', () => {
+  let component: RecommendationFormComponent;
+  let fixture: ComponentFixture<RecommendationFormComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ RecommendationFormComponent ],
+      imports: [
+        ReactiveFormsModule
+      ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RecommendationFormComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create the recommendation form component', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should render heading', () => {
+    const fixture = TestBed.createComponent(RecommendationFormComponent);
+    fixture.detectChanges();
```

Comment:
From what I understand, you don't need a fixture.detectChanges() in your test unless you're making a change in the fixture specifically for the test. Here, there will be nothing before this detectChange and one has already been made before in the beforeEach, so you might be able to omit it? (I'm not 100% sure!)

Reply:
Removed thanks :)